src/pages/bridge/helpers.ts

```diff
@@ -7,3 +15,36 @@ export function prioritizeXLMAsset(assets?: Asset[]): Asset[] {
 
   return assets.sort((a, b) => (a.code === 'XLM' ? -1 : b.code === 'XLM' ? 1 : 0));
 }
+
+const isInvalid = (value: unknown) => {
+  return !value;
+};
+
+export const useCalculateGriefingCollateral = (amount: Big, bridgedAsset: Asset | undefined) => {
```

Comment:
I think we should separate it to another file

Reply:
Done ✅